src/wallet/node_rpc_proxy.cpp

```diff
@@ -238,13 +239,24 @@ boost::optional<std::string> NodeRPCProxy::get_dynamic_base_fee_estimate(uint64_
     m_dynamic_base_fee_estimate = resp_t.fee;
     m_dynamic_base_fee_estimate_cached_height = height;
     m_dynamic_base_fee_estimate_grace_blocks = grace_blocks;
+    m_dynamic_base_fee_estimate_vector = std::move(resp_t.fees);
```

Comment:
Suggestion:

```
m_dynamic_base_fee_estimate_vector = !resp_t.fees.empty() ? std::move(resp_t.fees) : std::vector<uint64_t>{m_dynamic_base_fee_estimate};
```

Fixes a seg fault when wallet tries to construct a transaction before the fork (tries to access `fees[0]` on an empty vector in `get_dynamic_base_fee_estimate` below)